utils/setupClass.php

```diff
@@ -0,0 +1,844 @@
+<?php
+

```

Comment:
A namespace would be good. t least Nominatim (see the other classes), maybe even Nominatim\Setup.